model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -165,6 +165,17 @@ object AddressResponseAddress {
       underlyingScore = other.score,
       underlyingMaxScore = maxScore
     )
+  }
+
+  private def generateFormattedAddress(paf: PostcodeAddressFileAddress):String = {
+
+    val poBoxNumber = if (paf.poBoxNumber.isEmpty) "" else s"PO BOX ${paf.poBoxNumber}"
```

Comment:
The logic is following: If it is empty, then the text is empty, else prepend the text with `PO BOX `.